pkg/scheduler/framework/plugins/interpodaffinity/filtering.go

```diff
@@ -330,41 +330,40 @@ func getPreFilterState(cycleState *framework.CycleState) (*preFilterState, error
 
 // Checks if scheduling the pod onto this node would break any anti-affinity
 // terms indicated by the existing pods.
-func satisfiesExistingPodsAntiAffinity(state *preFilterState, nodeInfo *framework.NodeInfo) bool {
-	topologyMap := state.topologyToMatchedExistingAntiAffinityTerms
-
-	// Iterate over topology pairs to get any of the pods being affected by
-	// the scheduled pod anti-affinity terms
-	for topologyKey, topologyValue := range nodeInfo.Node().Labels {
-		if topologyMap[topologyPair{key: topologyKey, value: topologyValue}] > 0 {
-			return false
+func satisfyExistingPodsAntiAffinity(state *preFilterState, nodeInfo *framework.NodeInfo) bool {
```

Comment:
s/Pods/Pod/ to be consistent with others?

Reply:
this one should stay because we are checking existing pods (multiple pods), in the other two we are checking the incoming pod (single pod).